Add basic documentation pages (#988)
AI-Hypercomputer/maxtext: docs/getting_started/full_finetuning.md

# Full Finetuning LLama3-8B  Model

> In the pre-training section you saw the steps on how to do pre-training with
> MaxText. To perform full fine tuning, you need to pass the checkpoint to the
> training script. 

Author: ```suggestion
MaxText can perform pre-training and full finetuning. To perform full fine tuning, you need to pass the checkpoint to the training script. 
```
